tests/test_config.py

```diff
@@ -15,6 +15,40 @@
 
 
 SITE_YAML_FILENAME = "TESTONLY.site.yaml.bak"
+CONFIGURABLE_YAML = """
```

Comment:
What's the opinion on using a nested object here and writing it out to YAML later?

Reply:
This objects is not used at the moment, I added it for and example of intent long term. Happy to change the format or remove since it is not consumed yet.